brainstormforce/astra
# AST-2983 - If our Astra pro modules are off then in off canvas menu section showing extra separator

### Description

- If our Astra pro modules are off then in off canvas menu section showing extra separator
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2983

### Screenshots

https://d.pr/i/TBnMII

### Types of changes


- fix: divider readjusted for Submenu Colors section in customizer



### How has this been tested?

- Go to Customizer.
- Go to Offcanvas menu settings.
- Turn off addon Pro modules.
- Refresh and check divider for Submenu Colors design section, it should be placed properly.

### Checklist:
